docs/recipes/amp-alertmanager-terraform.md

```diff
@@ -0,0 +1,141 @@
+# Terraform as Infrastructure as a Code to deploy Amazon Managed Service for Prometheus and configure Alert manager
+
+In this recipe, we will demonstrate how you can use [Terraform](https://www.terraform.io/) to provision [Amazon Managed Service for Prometheus](https://aws.amazon.com/prometheus/) and configure rules management and alert manager to send notification to a [SNS](https://docs.aws.amazon.com/sns/) topic if a certain condition is met.
+
+
+!!! note
+    This guide will take approximately 30 minutes to complete.
+
+## Prerequisites
+You will need the following to complete the steps in this blog post:
+* [Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) 
+* [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+* [Terraform CLI](https://www.terraform.io/downloads)
+* [AWS Distro for Opentelemetry](https://aws.amazon.com/blogs/mt/amazon-managed-service-for-prometheus-is-now-generally-available/)
+* [eksctl](https://eksctl.io/)
+* [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html)
+* [jq](https://stedolan.github.io/jq/download/%20)
+* [helm](https://www.eksworkshop.com/beginner/060_helm/helm_intro/install/index.html)
+* [SNS topic]
+
+In the recipe, we will use a sample application in order to demonstrate the metric scraping using ADOT and remote write the metrics to the workspace. Fork and clone the sample app from the repository at aws-otel-community (https://github.com/aws-observability/aws-otel-community). Execute the below commands to deploy application inside the EKS cluster. Make sure the terminal has access to the EKS cluster, follow the link (https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-connection/) to connect to EKS cluster
+
+This Prometheus sample app generates all 4 Prometheus metric types (counter, gauge, histogram, summary) and exposes them at the /metrics endpoint
+
+A health check endpoint also exists at /
+
+The following is a list of optional command line flags for configuration:
+
+listen_address: (default = 0.0.0.0:8080)this defines the address and port that the sample app is exposed to. This is primarily to conform with the test framework requirements.
+metric_count: (default=1) the amount of each type of metric to generate. The same amount of metrics is always generated per metric type.
+label_count: (default=1) the amount of labels per metric to generate.
+datapoint_count: (default=1) the number of data-points per metric to generate.
+
+### Enabling Metric collection using AWS Distro for Opentelemetry
+1. Fork and clone the sample app from the repository at aws-otel-community.
+Then run the following commands.
+
+```
+cd ./sample-apps/prometheus
+docker build . -t prometheus-sample-app:latest
+```
+2. Push this image to a registry such as Amazon ECR or DockerHub.
```

Comment:
Let's focus on one, say, ECR and provide the commands. You can reuse, for example, https://aws-observability.github.io/aws-o11y-recipes/recipes/fargate-eks-metrics-go-adot-ampamg/#build-container-image